app/views/katello/api/v2/content_views/_content_view.json.rabl

```diff
@@ -22,8 +22,14 @@ child :environments => :environments do
   end
 end
 
-child :repositories => :repositories do
-  extends 'katello/api/v2/repositories/show'
+if @object.composite?
+  child :component_repositories => :repositories do
+    extends 'katello/api/v2/repositories/show'
```

Comment:
That is what it was doing before, which would be a valid change, but not one I think should be bundled with this.